--- orig/libretro-buildbot-recipe.sh	2022-04-27 20:24:28.878028740 +0000
+++ libretro-super/libretro-buildbot-recipe.sh	2022-04-27 20:27:45.458218787 +0000
@@ -649,6 +649,7 @@
 
 	[ "${ENABLED}" != "YES" ] && { echo "${NAME} is disabled, skipping"; continue; }
 
+	echo "::group::$NAME"
 	echo "buildbot job started at: $(date)"
 	echo
 	echo "buildbot job: $jobid processing $NAME"
@@ -755,6 +756,7 @@
 
 	echo
 	echo "buildbot job finished at: $(date)"
+	echo "::endgroup::"
 
 	cd "${BASE_DIR}"
 
@@ -794,6 +796,7 @@
 			echo "URL: $URL"
 			echo "REPO TYPE: $TYPE"
 			echo "ENABLED: $ENABLED"
+			BRANCH="miyoomini-1.10.3"
 
 			if [ -d "${PARENTDIR}/${DIR}/.git" ]; then
 				cd $PARENTDIR
@@ -1685,7 +1688,8 @@
 
 		echo 'building...'
 		echo "BUILD CMD: ${HELPER} ${MAKE} -j${JOBS}"
-		${HELPER} ${MAKE} -j${JOBS} 2>&1 | tee -a "$LOGFILE"
+		${HELPER} ${MAKE} -f Makefile.miyoomini -j${JOBS} 2>&1 | tee -a "$LOGFILE"
+		${HELPER} ${MAKE} -f Makefile.miyoomini_ml -j${JOBS} 2>&1 | tee -a "$LOGFILE"
 
 		status=$?
 		echo "$status"
@@ -1696,6 +1700,9 @@
 			MESSAGE="retroarch:	[status: done] [$jobid]"
 			echo buildbot job: $MESSAGE >> "$LOGFILE"
 			echo "Packaging"
+			mkdir dist
+			cp retroarch retroarch.cfg dist/
+			ls -lah dist/
 		else
 			MESSAGE="retroarch:	[status: fail] [$jobid]"
 			echo buildbot job: $MESSAGE >> "$LOGFILE"
